### ISSUE TITLE
[boosty.to] Site request for boosty.to (branch of ok.ru)

### ISSUE THREAD

**Issue by sideloading** [yt-dlp/yt-dlp#4212]
### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2022.06.22.1** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/ytdl-org/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

Global

### Example URLs

- Single video: https://boosty.to/vitaly/posts/acd4db50-18a8-40a0-a0bc-3bdee2022dbe 

### Description

This site uses ok.ru domain as seen in the verbose log (Odnoklassniki/ok.ru).

### Verbose log

```shell
[debug] Command-line config: ['-v', 'https://boosty.to/vitaly/posts/acd4db50-18a8-40a0-a0bc-3bdee2022dbe']
DeprecationWarning: Support for Python version 3.6 has been deprecated and will break in future versions of yt-dlp! Please update to Python 3.7 or above
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out UTF-8, error UTF-8, screen UTF-8
[debug] yt-dlp version 2022.06.22.1 [a86e01e] (zip)
[debug] Python version 3.6.6 (CPython 64bit) - Linux-5.4.0-113-generic-x86_64-with-debian-bullseye-sid
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.9.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-9.1
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://boosty.to/vitaly/posts/acd4db50-18a8-40a0-a0bc-3bdee2022dbe
[generic] acd4db50-18a8-40a0-a0bc-3bdee2022dbe: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] acd4db50-18a8-40a0-a0bc-3bdee2022dbe: Downloading webpage
[generic] acd4db50-18a8-40a0-a0bc-3bdee2022dbe: Extracting information
[debug] Looking for video embeds
[debug] [Odnoklassniki] Extracting URL: https://ok.ru/videoembed/3696046058009?sig=1YBGSjbeQuc&amp;uid=580692047577&amp;t=1656408449782&amp;cip=XXX.XXX.XXX.XX&amp;ua=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F90.0.4430.85+Safari%2F537.36&amp;promo=boosty
[Odnoklassniki] 3696046058009: Downloading desktop webpage
[Odnoklassniki] 3696046058009: Downloading mobile webpage
ERROR: [Odnoklassniki] 3696046058009: The author of this video has not been found or is blocked
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 647, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/odnoklassniki.py", line 179, in _real_extract
    raise e
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/odnoklassniki.py", line 173, in _real_extract
    return self._extract_desktop(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/odnoklassniki.py", line 195, in _extract_desktop
    raise ExtractorError(error, expected=True)
```

**Comment by gamer191** [comment]
> ```shell
> DeprecationWarning: Support for Python version 3.6 has been deprecated and will break in future versions of yt-dlp! Please update to Python 3.7 or above
> ```

Off topic, but I'd highly advise switching to the Linux standalone binaries, available at https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_linux, to save yourself the time of updating python

**Comment by phant** [comment]
Log from windows and yt-dlp version 2022.08.08

```
 .\yt-dlp.exe -v https://boosty.to/ikakprosto/posts/c040ff9d-9ece-4edb-b260-a7dc54a15e38
[debug] Command-line config: ['-v', 'https://boosty.to/ikakprosto/posts/c040ff9d-9ece-4edb-b260-a7dc54a15e38']
[debug] Encodings: locale cp1251, fs utf-8, pref cp1251, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.08.08 [3157158] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0.1-full_build-www.gyan.dev (setts), ffprobe 5.0.1-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://boosty.to/ikakprosto/posts/c040ff9d-9ece-4edb-b260-a7dc54a15e38
[generic] c040ff9d-9ece-4edb-b260-a7dc54a15e38: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] c040ff9d-9ece-4edb-b260-a7dc54a15e38: Extracting information
[debug] Looking for Brightcove embeds
[debug] Looking for embeds
[debug] Identified a Odnoklassniki embed
[debug] [Odnoklassniki] Extracting URL: https://ok.ru/videoembed/3808461007611?sig=tB4uieYcUBc&uid=580692047577&t=1660453980415&cip=XXX.XXX.XXX.X&ua=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F93.0.4577.63+Safari%2F537.36&promo=boosty
[Odnoklassniki] 3808461007611: Downloading desktop webpage
[Odnoklassniki] 3808461007611: Downloading mobile webpage
ERROR: [Odnoklassniki] 3808461007611: The author of this video has not been found or is blocked
  File "yt_dlp\extractor\common.py", line 666, in extract
  File "yt_dlp\extractor\odnoklassniki.py", line 171, in _real_extract
  File "yt_dlp\extractor\odnoklassniki.py", line 165, in _real_extract
  File "yt_dlp\extractor\odnoklassniki.py", line 187, in _extract_desktop
```

**Comment by megapro17** [comment]
https://github.com/megapro17/yt-dlp/commit/7f4007b0d7c27b538525622cb348d0f685f00244
```
python -m yt_dlp -F https://boosty.to/vitaly/posts/acd4db50-18a8-40a0-a0bc-3bdee2022dbe
[generic] acd4db50-18a8-40a0-a0bc-3bdee2022dbe: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] acd4db50-18a8-40a0-a0bc-3bdee2022dbe: Extracting information
[Odnoklassniki] 3696046058009: Downloading videoembed webpage
[Odnoklassniki] 3696046058009: Downloading m3u8 information
[info] Available formats for 3696046058009:
ID       EXT RESOLUTION FPS │   FILESIZE   TBR PROTO │ VCODEC    VBR ACODEC  ABR
────────────────────────────────────────────────────────────────────────────────
mobile   mp4 unknown        │                  https │ unknown       unknown
hls-214  mp4 256x144     30 │ ~ 27.63MiB  214k m3u8  │ unknown  214k unknown  0k
lowest   mp4 unknown        │                  https │ unknown       unknown
hls-433  mp4 426x240     30 │ ~ 55.84MiB  433k m3u8  │ unknown  433k unknown  0k
low      mp4 unknown        │                  https │ unknown       unknown
hls-871  mp4 640x360     30 │ ~112.33MiB  871k m3u8  │ unknown  871k unknown  0k
sd       mp4 unknown        │                  https │ unknown       unknown
hls-1441 mp4 852x480     30 │ ~185.76MiB 1441k m3u8  │ unknown 1441k unknown  0k
hd       mp4 unknown        │                  https │ unknown       unknown
hls-3064 mp4 1280x720    30 │ ~394.97MiB 3064k m3u8  │ unknown 3064k unknown  0k
full     mp4 unknown        │                  https │ unknown       unknown
hls-6467 mp4 1920x1080   30 │ ~833.65MiB 6467k m3u8  │ unknown 6467k unknown  0k
```
is there better way? links like https://ok.ru/videoembed/3696046058009 will not work

**Comment by Murtaught** [comment]
@megapro17 's fork works. Until it is merged into **yt-dlp** proper you can clone their repo and use it to download videos from boosty.

Use `--cookies-from-browser BROWSER` arg to download paid videos by authorizing with your account.
